cfme/common/provider.py

```diff
@@ -787,3 +777,11 @@ def import_all_modules_of(loc):
     path = project_path.join('{}'.format(loc.replace('.', '/'))).strpath
     for _, name, _ in pkgutil.iter_modules([path]):
         importlib.import_module('{}.{}'.format(loc, name))
+
+
+@navigator.register(CloudInfraProvider, 'AllNodes')
```

Comment:
I think calling this 'AllNodes' is misleading, it should be 'ProviderNodes' or something similar.